exercise-001/main.cpp

```diff
@@ -3,32 +3,48 @@
 
 #include "CLI/CLI.hpp"
 #include "config.h"
-
+#include "vector"
+#include "random"
+//.
 auto main(int argc, char **argv) -> int
 {
     /**
      * CLI11 is a command line parser to add command line options
      * More info at https://github.com/CLIUtils/CLI11#usage
      */
+    int count = 20;
     CLI::App app{PROJECT_NAME};
     try
     {
         app.set_version_flag("-V,--version", fmt::format("{} {}", PROJECT_VER, PROJECT_BUILD_DATE));
+
+        app.add_option("-c,--count", count, "A count Integer");       
         app.parse(argc, argv);
     }
     catch (const CLI::ParseError &e)
     {
         return app.exit(e);
     }
 
+std::vector<int> numbers;
+
+for(int i = 0; i<count; i++){
+    numbers.push_back(rand() % 100);
```

Comment:
```suggestion
    numbers.push_back((rand() % 100)+1);
```

To have random numbers in the range between 1..100